### DISCUSSION TITLE
how to stop mitmproxy interfering dns requests

### DISCUSSION THREAD

**Discussion by ariyanjm** [mitmproxy/mitmproxy#6198] [Q&A]
Hi guys 

i need to handle dns requests on my local machine(127.0.0.1) but when i lunch mitmproxy my dns server connection becomes irresponsive i winder if there is a whey that can have my local web server on port 443 and my local dns server on port 53 free running alongside mitmproxy

**Comment by ThomasBucaioni** [comment]
Hello,
Maybe you can put the proxy on another machine, something like this: https://docs.mitmproxy.org/stable/concepts-modes/#a-custom-gateway ?
